peer/network.go

```diff
@@ -335,8 +339,8 @@ func (n *network) AppRequest(ctx context.Context, nodeID ids.NodeID, requestID u
 
 	var req message.Request
 	if _, err := n.codec.Unmarshal(request, &req); err != nil {
-		log.Debug("failed to unmarshal app request", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request), "err", err)
-		return nil
+		log.Debug("forwarding unknown AppRequest to sdk", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request), "err", err)
```

Comment:
Shouldn't this log be an unmarshal error unlike the other changes (where it is "unknown")

Reply:
In this instance the SDK requests are prepended with a varint identifier, so they won't marshal correctly into the EVM's codec. It's possible that some opaque bytes we can't unmarshal with the EVM's codec came from the SDK, since the SDK handlers will define their own way of interpreting the bytes.